app/src/main/res/layout/addfeed.xml

```diff
@@ -87,87 +87,57 @@
                     android:textColor="?android:attr/textColorPrimary"/>
 
             <TextView
-                    android:id="@+id/btn_add_via_url"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:drawablePadding="8dp"
-                    app:drawableStartCompat="?attr/feed"
-                    app:drawableLeftCompat="?attr/feed"
-                    android:paddingTop="8dp"
-                    android:paddingBottom="8dp"
-                    android:background="?android:attr/selectableItemBackground"
-                    android:textColor="?android:attr/textColorPrimary"
-                    android:clickable="true"
-                    android:text="@string/add_podcast_by_url"/>
+                android:id="@+id/btn_add_via_url"
```

Comment:
Please keep the indention of the untouched lines the same

Reply:
Sorry, I didn't notice that. It's fixed now.

Reply:
Hmm, now the files are changed even more than before. To make it easier to read what your commit actually does, please keep the original formatting of the files, so that only the lines you actually changed are displayed on the "Files changed" tab.